moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp

```diff
@@ -121,7 +121,7 @@ OMPLInterface::getPlanningContext(const planning_scene::PlanningSceneConstPtr& p
 
 void OMPLInterface::configureContext(const ModelBasedPlanningContextPtr& context) const
 {
-  context->simplifySolutions(simplify_solutions_);
+//  context->simplifySolutions(simplify_solutions_);
```

Comment:
This should be better now. I didn't erase the `configureContext` function because it could be used for others things than to set `simplify_solutions_`, but it's empty now and could be remove.

Reply:
If a method like that might become useful again at some point we might reintroduce it, but it's a protected method of a standalone class, so please remove it as well.